HLshell/mycmd_1.py

```diff
@@ -1,24 +1,23 @@
 #!/usr/bin/env python
 import sys, os
 import pickle
-import argparse
-
-import threading
-import socket
-#import msvcrt
-import threading
-
+import time
 sys.path.append(r'../alicptfts')
 sys.path.append(r'./alicptfts')
-#sys.path.append(r'../../alicpt_workspace/alicptfts')
-#sys.path.append(r'../alicpt_workspace/alicptfts')
 
-from io import StringIO
-from functools import wraps
 
 from cmd import Cmd
 from re import match
-from alicptfts import AlicptFTS, FTSState
+import socket
+from io import StringIO
+from functools import wraps
+import argparse
+
+import msvcrt
```

Comment:
If I remember correctly, the GCS system runs on Centos.
The program need to be cross-platform or only based on Linux-like system

Reply:
Ah, looks like msvcrt doesn't work on linux. Since msvcrt is only used on the client side (which will be a windows computer), we can maybe put the import function in an os system check?